index.js

```diff
@@ -217,15 +230,19 @@ function adapter(uri, opts){
 
     debug('removing %s from rooms %s', id, Object.keys(rooms));
 
-    Adapter.prototype.delAll.call(this, id);
```

Comment:
However we called del() function repeatedly rooms.
https://github.com/GannettDigital/socket.io-ioredis/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR240
And del sids resource.
https://github.com/GannettDigital/socket.io-ioredis/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR242
Those sequence is totally as same as Adapter's delAll() function.
So we don't have to call Adapter's delAll() function.
This implementation like socket.io-redis.
https://github.com/socketio/socket.io-redis/blob/master/index.js#L212-L220